src/core/event.ts

```diff
@@ -12,7 +12,7 @@ export function bind(target: EventTarget, event: string, listener: EventListener
 export function reset(): void {
   // Walk through existing list of bindings and remove them all
   for (let binding of bindings) {
-    (binding.target).removeEventListener(binding.event, binding.listener, binding.capture);
+    try { binding.target.removeEventListener(binding.event, binding.listener, binding.capture); } catch { /* do nothing */ }
```

Comment:
Add a comment about why this could fail. Also is this the only location you remove event listeners? If not, should other locations also get this error handling?

Reply:
Thanks, added a comment.

Reply:
Also, added it to binding events and also added a comment. All event bindings go through this central part, so we should be ok.